source/uri-options/uri-options.rst

```diff
@@ -194,10 +194,10 @@ pertaining to URI options apply here.
      - The maximum replication lag, in wall clock time, that a secondary can suffer and still be eligible for server selection
 
    * - minPoolSize
-     - positive integer
+     - non-negative integer
      - defined in the `Connection Pooling spec`_
      - required for drivers with connection pools
-     - The maximum number of clients or connections able to be created by a pool at a given time
+     - The number of connections the driver should create and main in the pool even when no operations are occurring 
```

Comment:
Wording is off here. "should create and keep in the pool" sounds more like what's intended.